BlueprintUI/Sources/Measuring/SizeConstraint.swift

```diff
@@ -65,14 +65,23 @@ extension SizeConstraint {
 
         /// The measurement is unconstrained in the given dimension.
         case unconstrained
+        
+        /// Creates a `SizeConstraint` with the provided value.
+        public init(_ value : CGFloat) {
```

Comment:
Can we keep this `internal`?

Reply:
![G1F1ECN](https://user-images.githubusercontent.com/327847/80552828-b989fe80-897c-11ea-90d8-f9bdb3f59809.png)